src/sessionTypes/auth0WebAuth.js

```diff
@@ -119,7 +119,13 @@ class Auth0WebAuth {
               reject(err);
             }
 
-            resolve(profile);
+            const formattedProfile = {
+              ...profile,
```

Comment:
Is there a `created_at` field that should be handled similarly?

Reply:
Not from auth0